course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -43,6 +43,10 @@
                 $scope.lookup(searchTerm);
             }
         };
+        $scope.roleselected = function(role_id, role_name){
```

Comment:
style nitpicks:
* js is supposed to prefer camelCase, right?
* the other methods are of the form verbNoun(), can we stick with that?